tox.ini

```diff
@@ -303,6 +303,7 @@ deps =
     external_botocore-botocore128: botocore<1.29
     external_botocore-botocore0125: botocore<1.26
     external_botocore: moto
+    external_botocore: docker
```

Comment:
What's the docker dependency for? Lambda/moto?

Reply:
Yes it's import inside the lambda invoke in moto.